src/tiled/scriptfileinfo.cpp

```diff
@@ -242,3 +242,74 @@ QString ScriptFileInfo::symLinkTarget(const QString &file) const
 {
     return QFileInfo(file).symLinkTarget();
 }
+
+QString ScriptFileInfo::cleanPath(const QString &file) const
+{
+    return QDir().cleanPath(file);
+}
+
+QString ScriptFileInfo::fromNativeSeparators(const QString &file) const
+{
+    return QDir().fromNativeSeparators(file);
+}
+
+QString ScriptFileInfo::toNativeSeparators(const QString &file) const
+{
+    return QDir().fromNativeSeparators(file);
+}
+
+bool  ScriptFileInfo::cd(const QString &file)
+{
+    return QDir().cd(file);
+}
+
+uint  ScriptFileInfo::count(const QString &file) const
+{
+    return QDir(file).count();
+}
+
+QString ScriptFileInfo::dirName(const QString &file) const
+{
+    return QDir(file).dirName();
+}
+
+bool  ScriptFileInfo::mkdir(const QString &file) const
+{
+    return QDir().mkdir(file);
+}
+
+bool  ScriptFileInfo::mkpath(const QString &file) const
+{
+    return QDir().mkpath(file);
+}
+
+bool  ScriptFileInfo::remove(const QString &file)
+{
+    return QDir().remove(file);
```

Comment:
This just removes a file, so let's call [`QFile::remove`](https://doc.qt.io/qt-5/qfile.html#remove-1) for it instead.

Reply:
sounds good.